Makefile

```diff
@@ -125,76 +125,77 @@ dev: dev-install dev-stop
+MINIDCOS_WEB_PORT ?= 443
+
+minidcos-create:
+	@ minidcos docker inspect \
+	      --cluster-id $(MINIDCOS_CLUSTER_ID) \
 	      > /dev/null 2> /dev/null || \
-	( dcos-docker create dcos_generate_config.sh \
-	      --transport $(DCOS_DOCKER_TRANSPORT) \
-	      --masters $(DCOS_DOCKER_MASTERS) \
-	      --agents $(DCOS_DOCKER_AGENTS) \
-	      --public-agents $(DCOS_DOCKER_PUBLIC_AGENTS) \
-	      --cluster-id $(DCOS_DOCKER_CLUSTER_ID) \
-	      --custom-volume $(DCOS_DOCKER_CUSTOM_VOLUME) \
-	      $(DCOS_DOCKER_OPTS) && \
-	  dcos-docker wait \
-	      --transport $(DCOS_DOCKER_TRANSPORT) \
-	      --cluster-id $(DCOS_DOCKER_CLUSTER_ID) \
+	( minidcos docker create dcos_generate_config.sh \
+	      --transport $(MINIDCOS_TRANSPORT) \
+	      --masters $(MINIDCOS_MASTERS) \
+	      --agents $(MINIDCOS_AGENTS) \
+	      --public-agents $(MINIDCOS_PUBLIC_AGENTS) \
+	      --cluster-id $(MINIDCOS_CLUSTER_ID) \
+	      --custom-volume $(MINIDCOS_CUSTOM_VOLUME) \
+	      $(MINIDCOS_OPTS) && \
+	  minidcos docker wait \
+	      --transport $(MINIDCOS_TRANSPORT) \
+	      --cluster-id $(MINIDCOS_CLUSTER_ID) \
 	      --skip-http-checks )
 
-dcos-docker-destroy:
+minidcos-destroy:
 	@ docker ps \
-	    --filter label=dcos-e2e-web-id=$(DCOS_DOCKER_CLUSTER_ID) \
+	    --filter label=dcos_e2e.cluster_id=$(MINIDCOS_CLUSTER_ID) \
+	    --filter label=dcos_e2e.web_port \
 	    --format '{{.ID}}' \
-	| xargs docker kill > /dev/null
-	@ dcos-docker destroy --cluster-id $(DCOS_DOCKER_CLUSTER_ID)
+	| while read id; do [ -z "$$id" ] || docker kill $$id; done > /dev/null
+	@ minidcos docker destroy --cluster-id $(MINIDCOS_CLUSTER_ID)
 
-dcos-docker-web: dcos-docker-create
+minidcos-web: minidcos-create
 	@ docker ps \
-	    --filter label=dcos-e2e-web-id=$(DCOS_DOCKER_CLUSTER_ID) \
-	    --filter label=dcos-e2e-web-port=$(DCOS_DOCKER_WEB_PORT) \
+	    --filter label=dcos_e2e.cluster_id=$(MINIDCOS_CLUSTER_ID) \
+	    --filter label=dcos_e2e.web_port=$(MINIDCOS_WEB_PORT) \
 	    --format '{{.ID}}' \
-	| xargs docker kill > /dev/null
+	| while read id; do [ -z "$$id" ] || docker kill $$id; done > /dev/null
 	@ docker run \
 	    --rm --detach \
-	    --publish $(DCOS_DOCKER_WEB_PORT):$(DCOS_DOCKER_WEB_PORT) \
-	    --name $(shell dcos-docker inspect --cluster-id $(DCOS_DOCKER_CLUSTER_ID) | \
+	    --publish $(MINIDCOS_WEB_PORT):$(MINIDCOS_WEB_PORT) \
+	    --name $(shell minidcos docker inspect --cluster-id $(MINIDCOS_CLUSTER_ID) | \
 	                   jq -r .Nodes.masters[0].docker_container_name | \
-	                   sed -re 's/-master-0/-web-$(DCOS_DOCKER_WEB_PORT)/') \
-	    --label dcos-e2e-web-id=$(DCOS_DOCKER_CLUSTER_ID) \
-	    --label dcos-e2e-web-port=$(DCOS_DOCKER_WEB_PORT) \
-	    alpine/socat TCP4-LISTEN:$(DCOS_DOCKER_WEB_PORT),bind=0.0.0.0,reuseaddr,fork,su=nobody \
-	                 TCP4:$(shell dcos-docker inspect --cluster-id $(DCOS_DOCKER_CLUSTER_ID) | \
+	                   sed -re 's/-master-0/-web-$(MINIDCOS_WEB_PORT)/') \
```

Comment:
on my mac, this only work if I use `gnu-sed`. With `sed` I get:

```
sed: illegal option -- r
```
Are you also using gnu-sed @urbanserj? I saw a thread where folks fixed it, but I was planning just to document that you need gnu-sed on mac. What do you think @urbanserj?

Reply:
Yes, I have gnu sed installed. Could you please remove `-r`, i.e. `s/-re/-e/`? it should make `Makefile` fully posix compatible.